eo-runtime/src/main/java/org/eolang/PhTraced.java

```diff
@@ -35,11 +35,13 @@
  * @since 0.36
  */
 @Versionized
+@SuppressWarnings("PMD.TooManyMethods")
 public final class PhTraced implements Phi {
 
     /**
      * Name of property that responsible for keeping max depth.
      */
+    @SuppressWarnings("PMD.LongVariable")
```

Comment:
@c71n93 why can't you just make the name shorter?

Reply:
@yegor256 Because this variable means "maximum cage recursion depth". As for me the shortest name for this variable is `MAX_CAGE_RECURSION`. However max length of variable is two words. Any two-word options don't seem informative enough to me (`MAX_RECURSION`, `MAX_CAGE`, `CAGE_RECURSION`).

Reply:
@c71n93 how about `RECURSION_THRESHOLD`? The name of the class already provides a lot of semantic: `PhTraced.RECURSION_THRESHOLD` -- sounds good.